src/hooks/useLearntQuestions/useLearntQuestions.utils.ts

```diff
@@ -0,0 +1,50 @@
+import { Question, Subject, LearntQuestions } from 'validators/subjects';
+
+type SubjectId = Subject['id'];
+export type QuestionId = Question['id'];
+
+type LearntQuestionsKey = `learnt-${string}`;
+
+export const getLearntQuestionsLocalStorageKey = (
+  subjectId: SubjectId,
+): LearntQuestionsKey => `learnt-${subjectId}`;
+
+export const getLearntQuestionsFromLocalStorage = (
+  key: LearntQuestionsKey,
+): LearntQuestions => {
+  const questions = localStorage.getItem(key);
```

Comment:
Let's add validation to this using yup.

Reply:
Done, however, I had to do this a little bit later, after parsing the json.